src/main/java/Duke.java

```diff
@@ -1,10 +1,227 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.lang.reflect.Array;
+import java.sql.SQLOutput;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
+
+    private ArrayList<Task> taskList;
+
+    public Duke() {
+        this.taskList = new ArrayList<>();
+    }
+    public static String botMessage(String message) {
+        String space = "    ";
+        String horizontalBar = "-------------------------------------------------";
+        return space + horizontalBar + "\n" + space + message + "\n" + space + horizontalBar;
+    }
+
+    public static final String greeting = "Hello, I'm Capt. Price! What can I do for you today, Sergeant?";
+
+    public static final String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    public static final String horizontalBar = "-------------------------------------------------";
+    public static final String exitGreeting = "Over and out. See you next mission!";
```

Comment:
These are some interesting greetings! :+1: Would you like to consider making the constant names all caps?